### ISSUE TITLE
Thanks, this is what I need, you save me lots of time!!!!!

### ISSUE THREAD

**Issue by zzycami** [K1rakishou/ComposeSubsamplingScaleImage#1]
RT!!!!

**Comment by K1rakishou** [comment]
It's not production ready yet, there are bugs left to fix!

**Comment by zzycami** [comment]
I have fork this and  reading the code, I will fix some if I can!  anyway, this give us good ideas to resolve problem like this! tks!!!

**Comment by K1rakishou** [comment]
Oh I see.

**Comment by nachtien** [comment]
yeah agreed!  Nice work @K1rakishou!  I hope one day _if you have time_ that you can allow one to draw on top of the image. I'd love to place some sort of composable on it similar to how [MapCompose](https://github.com/p-lr/MapCompose/#markers) draws markers on it. 

Great job, I've starred and am watching this library.
